packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -59,7 +59,10 @@ export interface ChatMessageActionFlyoutProps {
 export const ChatMessageActionFlyout = (props: ChatMessageActionFlyoutProps): JSX.Element => {
   const theme = useTheme();
   const messageReadByCount = props.messageReadBy?.length;
-  const messageReadByList: IContextualMenuItem[] | undefined = props.messageReadBy?.map((person) => {
+
+  const sortedMessageReadyByList = props.messageReadBy?.sort((a, b) => a.displayName.localeCompare(b.displayName));
```

Comment:
`.sort` will still mutate the original array here, you will need to clone the array first then call `.sort`:
```suggestion
  const sortedMessageReadyByList = [...(props.messageReadBy ?? [])].sort((a, b) => a.displayName.localeCompare(b.displayName));
```

Spread operator here will clone `[...props.messageReadBy]`, you could also use `Array.from(props.messageReadBy)` to clone